### ISSUE TITLE
Source Map Upload：There are no artifacts in this archive

### ISSUE THREAD

**Issue by Hello-GBY** [getsentry/sentry-javascript-bundler-plugins#427]
### Environment
MacOS   M1  12.6
node: v16.14.0

## What version are you running? Etc.
sentry-cli 2.21.2
Sentry 22.11.0
use self-hosted.

### Steps to Reproduce
Execute the following commands in the terminal:
sentry-cli releases -o My_Org  -p seedpro-web files 1.0.0 upload-sourcemaps ./dist/assets --url-prefix https://annotation.test.work/

### Expected Result
upload files to sentry successfully,  Source Maps Archive has all the files just uploaded

### Actual Result

I want Source Map Upload. But there is no corresponding map file on the server side.

> josie@Josies-MacBook-Pro annotation % sentry-cli releases -o My_Org  -p seedpro-web files 1.0.0 upload-sourcemaps ./dist/assets --url-prefix https://annotation.test.work/
> Found 18 files
> Analyzing 18 sources
> Analyzing completed in 0.05s
> Rewriting sources
> Rewriting completed in 0.352s
> Adding source map references
> Bundling files for upload... https://annotation.test.work/pointCloudQuickSearch.00809dc6.js.ma
> Bundling completed in 1.071s
> Bundled 18 files for upload
> Bundle ID: e1b847d3-4f7f-586a-958d-0bc98ef52247
> Optimizing completed in 0.024s
> Uploading completed in 3.682s
> Uploaded files to Sentry
> Processing completed in 0.134s
> File upload complete (processing pending on server)
> Organization: My_Org
> Project: seedpro-web
> Release: 1.0.0
> Dist: None
> Upload type: release bundle

> Source Map Upload Report
>  Minified Scripts
> https://annotation.test.work/calcMappingPoint.870d8a6f.js (sourcemap at calcMappingPoint.870d8a6f.js.map)
>  https://annotation.test..work/getCameraViewRange.318eed4d.js (sourcemap at getCameraViewRange.318eed4d.js.map)
> https://annotation.test.work/getSelectPoint.f81c9d37.js (sourcemap at getSelectPoint.f81c9d37.js.map)
> https://annotation.test.work/index.6f5aed55.js (sourcemap at index.6f5aed55.js.map)
> https://annotation.test.work/pcdLodLoader.a956e47f.js (sourcemap at pcdLodLoader.a956e47f.js.map)
> https://annotation.test.work/pointCloudQuickSearch.00809dc6.js (sourcemap at pointCloudQuickSearch.00809dc6.js.map)
>  https://annotation.test.work/preloadFrameResource.9504d3ea.js (sourcemap at preloadFrameResource.9504d3ea.js.map)
>  https://annotation.test.work/pyWorker.0ace4fcf.js (sourcemap at pyWorker.0ace4fcf.js.map)
> https://annotation.test.work/updatePoint.8dd67194.js (sourcemap at updatePoint.8dd67194.js.map)
> Source Maps
>  https://annotation.test.work/calcMappingPoint.870d8a6f.js.map
 >  https://annotation.test.work/getCameraViewRange.318eed4d.js.map
>  https://annotation.test.work/getSelectPoint.f81c9d37.js.map
>  https://annotation.test.work/index.6f5aed55.js.map
>  https://annotation.test.work/pcdLodLoader.a956e47f.js.map
>  https://annotation.test.work/pointCloudQuickSearch.00809dc6.js.map
> https://annotation.test.work/preloadFrameResource.9504d3ea.js.map
> https://annotation.test.work/pyWorker.0ace4fcf.js.map
>  https://annotation.test.work/updatePoint.8dd67194.js.map

oops! : (

![image](https://github.com/getsentry/sentry-javascript-bundler-plugins/assets/65537456/9acb724b-128e-408f-b0df-94fa74664856)
![image](https://github.com/getsentry/sentry-javascript-bundler-plugins/assets/65537456/812c7f1e-294c-4047-b355-52f764a83b6e)

What actually happened. Maybe a screenshot/recording? Maybe some logs?

**Comment by lforst** [comment]
Duplicate of https://github.com/getsentry/sentry-javascript-bundler-plugins/issues/386, https://github.com/getsentry/sentry-javascript-bundler-plugins/issues/360. Please refer to these issues. If you feel like this is a different issue, feel free to ping us here and we will reopen! :)

**Comment by Hello-GBY** [comment]
> #386、#360的重复。请参考这些问题。如果您觉得这是一个不同的问题，请随时在这里联系我们，我们将重新开放！:)

I've moved self-hosted sentry to the latest version 23.7.1。
"@sentry/cli": "^2.21.5",
"@sentry/tracing": "^7.80.1",
"@sentry/vue": "^7.80.1",
"@sentry/vite-plugin": "^2.10.1"


After packaging, the upload was successful, but there are still no artifacts


![image](https://github.com/getsentry/sentry-javascript-bundler-plugins/assets/65537456/3ffaa59c-463c-410e-b14f-beff56fee368)
# no artifacts
![image](https://github.com/getsentry/sentry-javascript-bundler-plugins/assets/65537456/b46febe2-4288-4f88-b807-b69c6481f6af)
![image](https://github.com/getsentry/sentry-javascript-bundler-plugins/assets/65537456/dd082e03-69e6-4f5e-9d75-6ab12f0277ce)

**Comment by Hello-GBY** [comment]
> Duplicate of #386, #360. Please refer to these issues. If you feel like this is a different issue, feel free to ping us here and we will reopen! :)


I've moved self-hosted sentry to the latest version 23.7.1。
"@sentry/cli": "^2.21.5",
"@sentry/tracing": "^7.80.1",
"@sentry/vue": "^7.80.1",
"@sentry/vite-plugin": "^2.10.1"


After packaging, the upload was successful, but there are still no artifacts


![image](https://github.com/getsentry/sentry-javascript-bundler-plugins/assets/65537456/3ffaa59c-463c-410e-b14f-beff56fee368)
# no artifacts
![image](https://github.com/getsentry/sentry-javascript-bundler-plugins/assets/65537456/b46febe2-4288-4f88-b807-b69c6481f6af)
![image](https://github.com/getsentry/sentry-javascript-bundler-plugins/assets/65537456/dd082e03-69e6-4f5e-9d75-6ab12f0277ce)

**Comment by lforst** [comment]
This is fine. You should have artifact bundles instead.
